src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -186,7 +184,7 @@
                                                     <div class="form-group form-inline">
                                                         <div class="form-group text-muted">
                                                             <p>
-                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name) %>:
+                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name + " (" + data.student.team + ", " + data.student.email + ")") %>:
                                                             </p>
```

Comment:
I'm sorry to disagree:
![image](https://cloud.githubusercontent.com/assets/7261051/8696226/2aaea372-2b1d-11e5-83cb-12a5a0b28fcf.png)